apps/www/components/layout/user-account-nav.tsx

```diff
@@ -15,22 +16,32 @@ import {
 import { UserAvatar } from "@/components/shared/user-avatar";
 
 interface UserAccountNavProps extends React.HTMLAttributes<HTMLDivElement> {
-  user: Pick<User, "name" | "image" | "email">;
+  user: {
+    username: string;
+    email: string;
+    imageUrl: string;
+  };
 }
 
 export function UserAccountNav({ user }: UserAccountNavProps) {
+  const { signOut } = useClerk();
+  const router = useRouter();
+
   return (
     <DropdownMenu>
       <DropdownMenuTrigger>
         <UserAvatar
-          user={{ name: user?.name || null, image: user?.image || null }}
```

Comment:
Missing this after i have logged inn. Have not tried to debugg it.
How it is with this PR
![image](https://github.com/projectx-codehagen/projectx/assets/24507211/693914bb-d2c1-444c-a39b-acab8c4deda6)

How it should be:

![image](https://github.com/projectx-codehagen/projectx/assets/24507211/947a6038-2747-4451-8b83-17c8b67606c6)

Reply:
Thanks for the screenshot, I'll debug it and fix it